src/rax.c

```diff
@@ -1270,6 +1270,7 @@ void raxStart(raxIterator *it, rax *rt) {
  * is a low level function used to implement the iterator, not callable by
  * the user. Returns 0 on out of memory, otherwise 1 is returned. */
 int raxIteratorAddChars(raxIterator *it, unsigned char *s, size_t len) {
+    if (len == 0) return 1;
```

Comment:
There is memmove() call later. Looks like we call this function with s=NULL and len=0. memmove() is not supposed to be called with a NULL buffer even len=0.